packages/compiler/testing/test-host.ts

```diff
@@ -60,7 +60,7 @@ function createTestCompilerHost(
         .map((x) => x.replace(`${path}/`, ""));
     },
 
-    async removeDir(path: string) {
+    async rm(path: string) {
```

Comment:
Not sure how much it matters in test-host, but looks like this wouldn't actually delete a file since it doesn't start with path + slash? I see we're already not taking the options into account here, though.

Reply:
right, need to follow the recursive flag then